src/main/java/com/kibbezero/extralife/ExtraLife.java

```diff
@@ -54,7 +60,7 @@ private void doClientStuff(final FMLClientSetupEvent event) {
     private void enqueueIMC(final InterModEnqueueEvent event)
     {
         // some example code to dispatch IMC to another mod
-        InterModComms.sendTo("examplemod", "helloworld", () -> { LOGGER.info("Hello world from the MDK"); return "Hello world";});
+        InterModComms.sendTo("extralife", "helloworld", () -> { LOGGER.info("Hello world from the MDK"); return "Hello world";});
```

Comment:
This isn't related to Donor Drive stuff, but saw the reason this log message wasn't showing up and changed it.